app/javascript/packages/document-capture/components/status-message.tsx

```diff
@@ -24,7 +24,7 @@ function StatusMessage({ status, className, children }: StatusMessageProps) {
   const role = status === Status.ERROR ? 'alert' : 'status';
 
   return (
-    <span role={role} className={classes}>
+    <span role={role} className={classes} aria-live="polite">
```

Comment:
Rather than adding `aria-live`, should we hardcode the `role` to `status` instead? `status` has implied `aria-live` so it feels a little strange to specify both.

>Elements with the role status have an implicit [`aria-live`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) value of `polite`

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role

>Setting `role="alert"` is equivalent to setting [`aria-live="assertive"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live)

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role

```suggestion
    <span role="status" className={classes}>
```

Reply:
That makes sense, but with this new context I think the change I made was ill advised. I'm going to revert it for now since the original intent of this code still seems valid.

Restated, I think this is still what we should be doing:
```
  const role = status === Status.ERROR ? 'alert' : 'status';
```